CMakeLists.txt

```diff
@@ -577,6 +578,23 @@ IF(NOT QT_ANDROID)
   ENDIF(MSYS)
   INCLUDE(${wxWidgets_USE_FILE})
 
+  # As of cmake 3.11.2, these libraries are missing in list despite that
+  # we looked for them. This is a nasty fix which might fail miserably.
+  # Assumption: All builds using GTK uses unicode and wxWidgets 3.0
+  IF(GTK2_FOUND)
+    if (DEFINED WEBVIEW_FOUND)
+      list(APPEND wxWidgets_LIBRARIES -lwx_gtk2u_webview-3.0)
+    endif()
+  ELSEIF(GTK3_FOUND)
+    list(APPEND wxWidgets_LIBRARIES -lwx_gtk3u_aui-3.0)
+    if (OPENGL_FOUND)
+      list(APPEND wxWidgets_LIBRARIES -lwx_gtk3u_gl-3.0)
+    endif()
+    if (DEFINED WEBVIEW_FOUND)
+      list(APPEND wxWidgets_LIBRARIES -lwx_gtk3u_webview-3.0)
+    endif()
+  ENDIF ()
+
```

Comment:
I wonder if this is really needed as the previous state using `COMPONENTS` was that everybody reporting the cmake mess before was happy. It is really ugly and will break so I would much prefer not to have it.

Reply:
Yes, it's ugly, agreed.  Just making some experiments now, and need to sleep. It's indeed a mess, perhaps some sully detail...

Reply:
BTW, webview was just a I-need-some-sleep mistake. Don't merge...

Reply:
This is just brokenI misjudged this mess. Closing my second attempt